docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  website:
+    build:
+      context: .
+      dockerfile: deploy/Dockerfile.website
+    container_name: enei-website
+    ports:
+      - "3333:3333"
+
+  website-nginx:
+    image: nginx:1-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ./deploy/nginx-website.conf:/etc/nginx/nginx.conf
```

Comment:
```suggestion
```
In production, nginx config doesn't need to be mounted as volumes. If we can create an image with this inside, it's perfect.

Reply:
Ok, it is now creating the image with it inside, instead of creating a volume